packages/compass-components/src/components/index-keys.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Badge, { Variant } from '@leafygreen-ui/badge';
+import { css } from '@leafygreen-ui/emotion';
+import Icon from '@leafygreen-ui/icon';
+import { spacing } from '@leafygreen-ui/tokens';
+
+type IndexDirection =
+  | -1
+  | 1
+  | '2d'
+  | '2dsphere'
+  | 'text'
+  | 'geoHaystack'
+  | number;
+
+type IndexKeysProps = {
+  keys: {
+    field: string;
+    value: IndexDirection;
+  }[];
+};
+
+const containerStyles = css({
+  display: 'flex',
+  gap: spacing[1],
+});
+
+const IndexDirectionIcon = ({ direction }: { direction: IndexDirection }) => {
+  return direction === 1 ? (
+    <Icon glyph="ArrowUp" />
```

Comment:
This looks big compared to the font, we probably want to use small size icon here?
```suggestion
    <Icon glyph="ArrowUp" size="small" />
```